src/python/crash_analysis/stack_parsing/stack_analyzer.py

```diff
@@ -505,15 +511,22 @@
     (UBSAN_INTEGER_OVERFLOW_REGEX, 'Integer-overflow'),
     (UBSAN_UNSIGNED_INTEGER_OVERFLOW_REGEX, 'Unsigned-integer-overflow'),
     (UBSAN_INVALID_BOOL_VALUE_REGEX, 'Invalid-bool-value'),
+    (UBSAN_INVALID_BUILTIN_REGEX, 'Invalid-builtin-use'),
     (UBSAN_MISALIGNED_ADDRESS_REGEX, 'Misaligned-address'),
     (UBSAN_NO_RETURN_VALUE_REGEX, 'No-return-value'),
     (UBSAN_NULL_POINTER_READ_REGEX, 'Null-dereference READ'),
     (UBSAN_NULL_POINTER_REFERENCE_REGEX, 'Potential-null-reference'),
     (UBSAN_NULL_POINTER_WRITE_REGEX, 'Null-dereference WRITE'),
     (UBSAN_OBJECT_SIZE_REGEX, 'Object-size'),
     (UBSAN_POINTER_OVERFLOW_REGEX, 'Pointer-overflow'),
+    (UBSAN_RETURNS_NONNULL_ATTRIBUTE_REGEX, 'Invalid-null-return'),
     (UBSAN_SHIFT_ERROR_REGEX, 'Undefined-shift'),
+    (UBSAN_UNREACHABLE_REGEX, 'Unreachable code'),
```

Comment:
Unreachable-code

Reply:
We use this type in other parts of the code. I'd prefer to be consistent.

Reply:
This is fine.